frappe/utils/response.py

```diff
@@ -191,7 +191,13 @@ def send_private_file(path):
 		response = Response(wrap_file(frappe.local.request.environ, f), direct_passthrough=True)
 
 	# no need for content disposition and force download. let browser handle its opening.
```

Comment:
Apparently, there was some thought once upon a time for doing this for every file.